traffic_ops/testing/api/v3/servers_test.go

```diff
@@ -248,6 +249,8 @@ func UpdateTestServers(t *testing.T) {
 
 	// Retrieve the server by hostname so we can get the id for the Update
 	resp, _, err := TOSession.GetServers(&params, nil)
+	originalHostname := *resp.Response[0].HostName
```

Comment:
Test cases to test the following:
1. Change in hostname updated the field hostName but doesn't impact xmppid
2. Also, any change to xmppid, the Update() ignores the change.

Also, the original hostname is retained to ensure all the following test cases pass.